### ISSUE TITLE
nf-aggregate downloads fusion logs when fusion is not enabled

### ISSUE THREAD

**Issue by ahvigil** [seqeralabs/nf-aggregate#18]
Running nf-aggregate workflow against our tower instance generates a massive number of errors similar to the following:
```
Jan-30 21:03:59.865 [Thread-119858] - ERROR i.s.t.service.platform.ssh.SSHUtils - Unable to transfer SCP file=/locus/data/nextflow/work/prd/66/0f3f946b36cc080e4331d090a69099/.fusion.log - Cause: Remote SCP command returned error: scp: /locus/data/nextflow/work/prd/66/0f3f946b36cc080e4331d090a69099/.fusion.log: No such file or directory
```
I think this is expected because we don't run our workflows with fusion currently. These missing file errors seem to be pretty expensive, which means my nf-aggregate run created a massive and sustained spike in load for our Seqera platform cluster over several hours. I was able to cancel and rerun with `--skip_run_gantt` to get rid of the errors and that subsequent attempt completed in just a few minutes without issue.

It seems the gantt chart generation significantly increases the footprint of this pipeline against a Seqera Platform instance, and expects workflows to be run with fusion enabled. Would be nice if nf-aggregate were a bit smarter about this, especially since this is the default behavior. I also don't see Fusion mentioned anywhere in the prerequisites or READMe, and I think its worth a mention.

**Comment by drpatelh** [comment]
Thank you for the feedback @ahvigil ! We will update the pipeline to be better for this use case before the next release.

**Comment by drpatelh** [comment]
Will be fixed in v0.2.0 via #30
